File: go-sdk/pkg/models/operations/queueadeployrequest.go
package operations

type QueueADeployRequestPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type QueueADeployRequestRequest struct {
	PathParams QueueADeployRequestPathParams
}

type QueueADeployRequestResponse struct {
	ContentType                                 string
	StatusCode                                  int64
	QueueADeployRequest200ApplicationJSONObject map[string]map[string]interface{}
}

File: ts-sdk/src/sdk/models/operations/queueadeployrequest.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Type } from "class-transformer";


export class QueueADeployRequestPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class QueueADeployRequestRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: QueueADeployRequestPathParams;
}

export class QueueADeployRequestResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  queueADeployRequest200ApplicationJSONObject?: Record<string, Record<string, any>>;
}